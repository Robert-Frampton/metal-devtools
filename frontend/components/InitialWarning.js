import Component from 'metal-jsx';

class InitialWarning extends Component {
	render() {
		return (
			<div class="initial-warning-container">
				<div>
					{
						'If you do not see your components here, try refreshing the page while keeping this panel open.'
					}
					<br />
					<i>
						{
							'Or there may be no Metal.js components on the page. metal@^2.6.2 required. '
						}
						<a
							href="https://github.com/bryceosterhaus/metal-devtools"
							target="blank_"
						>
							{'Repo'}
						</a>
					</i>
				</div>
			</div>
		);
	}
}

export default InitialWarning;
