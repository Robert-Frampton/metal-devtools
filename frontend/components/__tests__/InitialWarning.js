jest.unmock('../InitialWarning');

import InitialWarning from '../InitialWarning';

describe('InitialWarning', () => {
	test('should render', () => {
		const component = new InitialWarning();

		expect(snap(component)).toMatchSnapshot();
	});
});
