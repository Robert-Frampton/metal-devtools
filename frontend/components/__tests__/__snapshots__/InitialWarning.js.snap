exports[`InitialWarning should render 1`] = `
<div class="initial-warning-container">
  <div key="_metal_jsx_1">If you do not see your components here, try refreshing the page while keeping this panel open.
    <br key="_metal_jsx_1">
    <i key="_metal_jsx_2">Or there may be no Metal.js components on the page. metal@^2.6.2 required.
      <a href="https://github.com/bryceosterhaus/metal-devtools" target="blank_" key="_metal_jsx_1">Repo</a>
    </i>
  </div>
</div>
`;
